File: LoopConstruction/C++/NoGuessGame.cpp
#include <iostream>
#include <cstdlib>
using namespace std;

int main() {
    int guess, value;

    do {
        cout << "Enter your guess value between 1 to 10: ";
        cin >> guess;

        value = rand() % 10 + 1; // Generate random value between 1 to 10
        cout << "Generated Value is " << value << endl;

        if (guess > value) {
            cout << "You guessed a larger value." << endl;
        } else if (guess == value) {
            cout << "Congrats! Your guess is right." << endl;
            break;
        } else {
            cout << "You guessed a smaller value." << endl;
        }
    } while (true);

    return 0;
}

File: LoopConstruction/Java/NoGuessGame.java

import java.util.Scanner;

public class NoGuessGame {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

       do { 

        System.out.println("Enter your guess value between 1 to 10");
        int guess = sc.nextInt();

        int value = (int)(Math.random()*10 +1);
        System.out.println("Genretaed Value is "+value);

        if(guess>value){
            System.out.println("You guess Large value ");
        }else if(guess == value){
            System.out.println("Congrats Your guess is right");
            break;
        }else{
            System.out.println("You guess small value ");
        }
           
       } while (true);
    }
}
